src/components/swap/SwapBox.vue

```diff
@@ -183,6 +214,150 @@ export default defineComponent({
     toggleshowMaxTooltip() {
       this.showMaxTooltip = !this.showMaxTooltip;
     },
+    async onApprove() {
```

Comment:
this shouln't be here, we don't have approvals anymore.

Reply:
removed too